supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc

```diff
@@ -81,17 +81,28 @@
 
 *See also*:
 
-// Azure: General; kept as is
 [discrete]
 [[dotnet]]
-==== image:images/yes.png[yes] .NET Core (noun)
-*Description*: Microsoft's newest development platform, ".NET Core", is an open source and cross-platform framework for building cloud-based internet-connected applications, such as web apps, IoT apps, and mobile backends. All .NET Core applications can run on .NET Core or on the full .NET Framework. Always refer to it as .NET Core; all other variants are incorrect.
+==== image:images/caution.png[with caution] .NET (noun)
+*Description*: Microsoft's development platform, ".NET", is an open source and cross-platform framework for building cloud-based internet-connected applications, such as web apps, IoT apps, and mobile back ends. All .NET applications can run on .NET or on the .NET Framework. For version 5 and later, refer to it as ".NET". For earlier versions, use ".NET Core".
 
-*Use it*: yes
+*Use it*: with caution
 
-*Incorrect forms*: dotNet Core, .Net Core, .NET, .Net
+*Incorrect forms*: .Net, dotNet, .NET Core (incorrect for version >= 5), dotNet Core, .Net Core
 
-*See also*:
+*See also*: xref:dotnetcore[.Net Core]
+
+// Azure: General; kept as is
+[discrete]
+[[dotnetcore]]
+==== image:images/caution.png[with caution] .NET Core (noun)
+*Description*: Microsoft's development platform, ".NET Core", is an open source and cross-platform framework for building cloud-based internet-connected applications, such as web apps, IoT apps, and mobile back ends. All .NET Core applications can run on .NET Core or on the .NET Framework. For version 3.1 and earlier, refer to it as ".NET Core". In later versions, Microsoft changed the name to ".NET".
```

Comment:
This says 3.1 and earlier, and the other entry says version 5 and later. Just  checking that there aren't any .NET/.NET Core versions at all between 3.1 and 5?

Reply:
There was no version 4. See: https://en.wikipedia.org/wiki/.NET :
"In November 2020, Microsoft released .NET 5.0. The "Core" branding was removed and version 4.0 was skipped to avoid conflation with .NET Framework, which remains the Windows-specific product."

See also: https://dotnet.microsoft.com/en-us/download/dotnet

Reply:
Cool thanks for clarifying, just wanted to check! :slightly_smiling_face: